.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/forcedotcom/LightningTestingService
+    docker:
+      - image: circleci/node:10.15.3-browsers
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+      SFDX_AUTOUPDATE_DISABLE: true
+      SFDX_USE_GENERIC_UNIX_KEYCHAIN: true
+      SFDX_DOMAIN_RETRY: 300
+    steps:
+      - checkout
+      # Prepare for artifact and test results  collection equivalent to how it was done on 1.0.
+      # In many cases you can simplify this from what is generated here.
+      # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
+      - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
+      # Dependencies
+      #   This would typically go in either a build or a build-and-test job when using workflows
+      # Restore the dependency cache
+      - restore_cache:
+          keys:
+            # This branch if available
+            - v1-dep-{{ .Branch }}-
+            # Default branch if not
+            - v1-dep-master-
+            # Any branch if there are none on the default branch - this should be unnecessary if you have your default branch configured correctly
+            - v1-dep-
+      - run: sudo chown -R circleci /usr/local/lib/node_modules
+      - run: sudo npm install -g sfdx-cli
+      - run: sudo apt-get update
+      - run: sudo apt-get install vim-common
+      # Test
```

Comment:
would it make more sense to put this in a test job or something like that for clearer reporting?

Reply:
Whats a test job?

Reply:
I meant a separate job named "test". So, for example when you look at this https://github.com/forcedotcom/LightningTestingService/pull/99/checks?check_run_id=286591987 it shows build and test as two separate items